content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Jaeger
+description: How to configure the proxies to send tracing requests to Jaeger.
+weight: 10
+keywords: [telemetry,tracing,jaeger]
+---
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+    Either use the `istio-demo.yaml` or `istio-demo-auth.yaml` template, which includes tracing support, or
+    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option.
+
+* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Accessing the dashboard
+
+Setup access to the tracing dashboard by using port-forwarding:
+
+{{< text bash >}}
+$ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=istio-ingressgateway -o jsonpath='{.items[0].metadata.name}') 15032:15032 &
+{{< /text >}}
+
+Access the dashboard by opening your browser to [http://localhost:15032](http://localhost:15032).
+
+It is also possible to use a Kubernetes ingress by specifying the Helm chart option `--set tracing.ingress.enabled=true`.
```

Comment:
Do not use relative pronouns. The reference for the pronoun "It" is ambiguous. Remove the entire conditional sentence and make this a new step entirely.
```suggestion
1. To use a Kubernetes ingress, specify the Helm chart option `--set tracing.ingress.enabled=true`.
```